tests/apm_tracing_e2e/test_process_tags.py

```diff
@@ -1,12 +1,12 @@
 from utils import weblog, interfaces, scenarios, features, context
-from utils._decorators import missing_feature
+from utils._decorators import missing_feature, irrelevant
 from utils.interfaces._library.miscs import validate_process_tags
 
 
 @scenarios.tracing_config_nondefault
 @features.process_tags
 @missing_feature(
-    condition=context.library.name not in ("java", "golang") or context.weblog_variant == "spring-boot-3-native",
```

Comment:
for context this was ignored since the java settings have to be passed along the native image builder. Reason why it could not work. The reason why now is working for `1.53.0` and onward (`master`) is because we activated for java 21 (used by the spring boot native if not wrong) and for everybody in master